app/models/manageiq/providers/workflows/automation_manager/workflow_instance.rb

```diff
@@ -99,4 +102,21 @@ def run(zone: nil, role: "automation", object_type: nil, object_id: nil)
 
     run_queue(:zone => zone, :role => role, :object => object) if next_state.present?
   end
+
+  class << self
+    def floe_docker_runner
+      @floe_docker_runner ||= begin
+        if MiqEnvironment::Command.is_podified?
```

Comment:
This is interesting with respect to the monolithic image.  In that case, we might do `docker` and require the monolithic image be run with the docker socket?  Not sure what to do there...perhaps @bdunne has thoughts.  Since that's not a primary use case, though we can just `TODO` it for now, and I won't let that hold up merge.

Reply:
Actually that case should be handled, we have `is_container?` which covers the monolithic docker container and `is_podified?` checks that we are on k8s (it checks the `/run/secrets/kubernetes.io/serviceaccount/token` service account token)

Reply:
I picked `is_podified?` specifically to be k8s and not monolithic docker